sfm_pc/sql/areas.sql

```diff
@@ -2,11 +2,16 @@ SELECT
   object_ref.uuid AS uuid,
   MAX(name.value) AS name,
   MAX(division_id.value) AS division_id,
+  substring(MAX(division_id.value), position(':' IN MAX(division_id.value)) + 1, 2) AS country_iso,
```

Comment:
Great work on these functions 👏 For the purposes of this PR, I think this change works well. In the future, if we realize we need to make adjustments to these operations, then it might be worth creating [stored procedures](https://carto.com/help/working-with-data/sql-stored-procedures/#basic-structure-of-a-plpgsql-function) to modularize the functions. But for now, I think a little copy-pasting is advantageous because it means that all of the query logic is scoped to the individual SQL file that runs the query; we'll see in the future whether or not this creates a maintenance burden.